chia/cmds/wallet_funcs.py

```diff
@@ -68,12 +68,23 @@ async def get_transactions(args: dict, wallet_client: WalletRpcClient, fingerpri
                 break
 
 
+def check_unusual_transaction(amount: Decimal, fee: Decimal):
+    return fee >= amount
```

Comment:
this is not a very low threshold for the fee. you could send a very large fee this way. I suppose it's conservative. To come up with something stricter makes it difficult for small transactions.